.travis.yml

```diff
@@ -1,30 +1,45 @@
 os:
   - linux
-sudo: false
+dist: trusty
+language: android
+
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
     packages:
       - libstdc++6
       - fonts-noto
+
 git:
   depth: 3
+
+android:
+  components:
+    - build-tools-28.0.3
+    - build-tools-29.0.2
```

Comment:
q: is it intended to have two build-tool versions?

Reply:
Discussed offline, removed hardcoded version from the apps' build.gradle files.